src/BlazorWebView/src/Maui/iOS/IOSWebViewManager.cs

```diff
@@ -74,10 +75,20 @@ public override void DidStartProvisionalNavigation(WKWebView webView, WKNavigati
 
 			public override void DecidePolicy(WKWebView webView, WKNavigationAction navigationAction, Action<WKNavigationActionPolicy> decisionHandler)
 			{
-				if (navigationAction.TargetFrame!.MainFrame)
+				// TargetFrame is null for navigation to a new window (`_blank`)
```

Comment:
Could you clarify which of the following have `TargetFrame` being null?

 * A regular `<a>` link with no `target`
 * An `<a>` link with `target="_blank"` explicitly
 * Navigation via `location.href = 'https://externalsite.com';`

I found the comment to be a bit ambiguous because it mentions `_blank` but I'm unclear if developers are required to add that explicitly. If all three cases work with this implementation that's great.

Reply:
Actually on reading this more, I guess that it's only the second case (explicit `target="_blank"`). If that's true, it's totally reasonable for this PR.

Later on I think our goal from security review was to ensure that you can't navigate to external content in the app's webview by default, and that developers would explicitly have to configure or override something to allow that. Not saying that should be addressed in this PR - just checking our understandings align!

Reply:
> it's only the second case (explicit target="_blank"). If that's true, it's totally reasonable for this PR.

Exactly. In all other cases we would have a non-null target frame. This is in accordance with the [docs](https://docs.microsoft.com/en-us/dotnet/api/webkit.wknavigationaction.targetframe?view=xamarin-ios-sdk-12#remarks).

> Later on I think our goal from security review was to ensure that you can't navigate to external content in the app's webview by default, and that developers would explicitly have to configure or override something to allow that. Not saying that should be addressed in this PR - just checking our understandings align!

Makes sense, created https://github.com/dotnet/maui/issues/4338 to evaluate this further!

Reply:
I've been seeing an issue where attempting to open external URLs in a Blazor hybrid app for iOS causes crashes. Is this related by any chance? https://github.com/dotnet/maui/issues/3980#issuecomment-1030198360

Reply:
Yes this PR fixes the issue you're seeing in that comment.